elements/rh-card/docs/20-guidelines.md

```diff
@@ -19,6 +19,10 @@ see the [card patterns page](/patterns/card).
 
 </rh-alert>
 
+### Cards vs. tiles
+
+The main difference between a card and a [tile](/elements/tile/) is that cards can be used as content containers with or without links or other actions. Cards can also contain more than one calls to action, whereas tiles **must** perform only one action. Like tiles, however, cards can be grouped together.
```

Comment:
> more than one calls to action

Simple fix: delete the `s` in `calls`

Or, if you want to rework it a little more...


> The main difference between a card and a [tile](/elements/tile/) is that cards may or may not contain links or other actions. Cards can even contain multiple calls to action.
> 
> Tiles, on the other hand, **must** perform exactly one action. No more or less.
>
> Cards can be grouped together with other cards, and tiles can be grouped together with other tiles.